src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -758,6 +758,29 @@ default boolean isAutomaticModule(ModuleElement module) {
         return false;
     }
 
+    /**
+     * {@return the class body of an {@code enum} constant if the
+     * argument is an {@code enum} constant declared with an optional
+     * class body, {@code null} otherwise}
+     *
+     * @implSpec
+     * The default implementation of this method returns {@code null}
+     * if the argument is an {@code enum} constant and throws an
+     * {@code IllegalArgumentException}  if it is not.
+     *
+     * @throws IllegalArgumentException if the argument is not an {@code enum} constant
+     * @param enumConstant an enum constant
```

Comment:
nice, my only comment is that the order selected in the javadoc seems to be at odds with the rest of the javadoc in this class. The common order I see is:

1. @implSpec
2. @param
3. @return
4. @throws

Reply:
Tags reordered; thanks.